home/start_page.py

```diff
@@ -186,24 +185,17 @@ def __init__(self, app, allow_move=False, allow_manage=True):
 class CollapsableAppWidget(ipw.Accordion):
     """Widget that represents a collapsable app as part of the home page."""
 
-    hidden = traitlets.Bool()
+    hidden = traitlets.Bool(None, allow_none=True)
 
     def __init__(self, app, **kwargs):
         self.app = app
         app_widget = AppWidget(app, **kwargs)
         super().__init__(children=[app_widget])
         self.set_title(0, app.title)
-        # Need to observe all names here due to unidentified issue:
-        self.observe(
-            self._observe_accordion_selected_index
-        )  # , names=['selected_index'])
 
-    @_workaround_property_lock_issue
```

Comment:
Great! Please remove this decorator (defined in this file) since it is not used anywhere else.